### DISCUSSION TITLE
Error on the command "whisper --version"

### DISCUSSION THREAD

**Discussion by Jordan-CELESTIN** [openai/whisper#1502] [Q&A]
Hi, 
I installed whisper but I got this message after the commande "whisper --version": 

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\jorda\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\Scripts\whisper.exe\__main__.py", line 4, in <module>
  File "C:\Users\jorda\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\whisper.py", line 69, in <module>
    libc = ctypes.CDLL(libc_name)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1264.0_x64__qbz5n2kfra8p0\Lib\ctypes\__init__.py", line 366, in __init__
    if '/' in name or '\\' in name:
       ^^^^^^^^^^^
TypeError: argument of type 'NoneType' is not iterable

I don't know how to resolve it.

   Thanks for your suggestions !

**Comment by phineas-pta** [comment]
that command doesn't exist, try transcribe

**Reply by Jordan-CELESTIN** [reply]
It doesn't work unfortunately.
Maybe I did a wrong configuration or there is an incompatibility on Python or the dépendances use by whisper...
Pitorch version: 2.0.1+cu118
Python: 3.11.4

**Reply by phineas-pta** [reply]
can u show the command u use & full error message

**Reply by Jordan-CELESTIN** [reply]
![image](https://github.com/openai/whisper/assets/138792660/6dc005a5-a087-4c52-ba70-0319555e8ce9)

**Reply by phineas-pta** [reply]
did u install the correct whisper?

**Reply by Jordan-CELESTIN** [reply]
Maybe I made the wrong, can you give me the right one to see if that's the case ?
If yes, I'll uninstall and reinstall to try.

**Reply by phineas-pta** [reply]
what u see when run `pip show openai-whisper`

**Reply by abdelmajid-hachim** [reply]
j 'ai le même problème , si je tape la commande suivante : pip show openai-whisper

E:\projet python\whisper>pip show  openai-whisper
Name: openai-whisper
Version: 20231117
Summary: Robust Speech Recognition via Large-Scale Weak Supervision
Home-page: https://github.com/openai/whisper
Author: OpenAI
Author-email:
License: MIT
Location: c:\users\hachima\appdata\local\programs\python\python310\lib\site-packages
Requires: more-itertools, numba, numpy, tiktoken, torch, tqdm
Required-by:

merci de votre aide

**Reply by phineas-pta** [reply]
ton truc est pété

il faut réinstaller

**Comment by Jordan-CELESTIN** [comment]
It's the right one.
I'll try unintall and reinstall tomorrow


![image](https://github.com/openai/whisper/assets/138792660/a8ccb445-392c-4634-afa4-ef9ddadff48d)

**Comment by Jordan-CELESTIN** [comment]
I just did it and it work properly now.
I probably had incompatibles version. 
Thanks for the help!
